rep-2000.rst

```diff
@@ -547,6 +547,126 @@ Build System Support:
+
+
+Dependency Requirements:
+
++-------------+-------------------------------------------------+--------------------------------------+
+|             |                        Required Support         |        Recommended Support           |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Package     |  Ubuntu Focal  |     MacOS**   |   Windows 10** | Debian Buster  | OpenEmbedded**      |
++=============+================+===============+================+================+=====================+
+| CMake       |     3.16.3     |     3.14.4    |     3.14.4     |      3.13.4    | 3.16.1 / 3.12.2**** |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| EmPY        |                                      3.3.2                                             |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Gazebo      |     11.0.0*    |     11.0.0    |      N/A       |      11.0.0*   |        N/A          |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Ignition    |             Citadel*           |      N/A       |     Citadel8   |        N/A          |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Ogre        |                                      1.10*                       |        N/A          |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| OpenCV      |     4.2.0      |     4.2.0     |     3.4.6*     |      3.2.0     |  4.1.0 / 3.2.0****  |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| OpenSSL     |     1.1.1d     |     1.1.1f    |     1.1.1f     |      1.1.1d    | 1.1.1d / 1.1.1b**** |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Poco        |     1.9.2      |     1.9.0     |     1.8.0*     |      1.9.0     |        1.9.4        |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Python      |     3.8.0      |     3.7.3     |     3.8.0      |      3.7.3     |  3.8.2 / 3.7.5****  |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| Qt          |     5.12.5     |     5.12.3    |     5.10.0     |      5.11.3    | 5.14.1 / 5.12.5**** |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+|                              |         **Linux only**                                                |
++-------------+----------------+---------------+----------------+----------------+---------------------+
+| PCL         |     1.10.0     |     N/A       |     N/A        |      1.9.1     |        1.8.1        |
```

Comment:
PCL v1.10.0 will be provided by OpenEmbedded. I've created a PR to update it [here](https://github.com/ros-infrastructure/rep/pull/249).